src/Concordium/Client/Commands.hs

```diff
@@ -913,6 +1002,7 @@ accountEncryptCmd =
             ( AccountEncrypt
                 <$> transactionOptsParser
                 <*> option (eitherReader amountFromStringInform) (long "amount" <> metavar "CCD-AMOUNT" <> help "The amount to transfer to shielded balance.")
+                <*> optional (strOption (long "outFile" <> metavar "FILE" <> help "An optional file name and path to ouptput the signed/partially-signed transaction to instead of submitting the transaction on-chain."))
```

Comment:
I think this is not needed since it is already in `transactionOptsParserBuilder` used in `transactionOptsParser` :)